src/app/components/about/about.component.html

```diff
@@ -0,0 +1,17 @@
+<h2>About Cristhian Castro</h2>
+<article>
+  <div class="text">
+    <h3>Just the highlights</h3>
+    <p class="blackbox">
+      Experienced Bolivian Software Engineer, with great interest in
+      development. I like to help businesses, bringing them high tech solutions.
+    </p>
+    <h4>My favorite technologies include:</h4>
+    <ul>
```

Comment:
Would be interesting to add this to your test using data-test-id attributes:
https://betterprogramming.pub/decouple-tests-with-data-attributes-c920606c5f27